ethcore/src/ethereum/ethash.rs

```diff
@@ -73,6 +75,7 @@ impl From<ethjson::spec::EthashParams> for EthashParams {
 			difficulty_hardfork_transition: p.difficulty_hardfork_transition.map_or(0x7fffffffffffffff, Into::into),
 			difficulty_hardfork_bound_divisor: p.difficulty_hardfork_bound_divisor.map_or(p.difficulty_bound_divisor.into(), Into::into),
 			bomb_defuse_transition: p.bomb_defuse_transition.map_or(0x7fffffffffffffff, Into::into),
+			bad_gas_compatibility_mode_limit: p.bad_gas_compatibility_mode_limit.map_or(0, Into::into),
```

Comment:
yeah. the thing is that if we don't default it to 'on', then all private chains will be using a broken protocol and run the risk of being internally DoSed.

we can print a warning to ensure they know what went wrong.